src/org/opensolaris/opengrok/authorization/AuthorizationFramework.java

```diff
@@ -452,15 +452,15 @@ private IAuthorizationPlugin loadClass(String classname) throws ClassNotFoundExc
      */
     private void loadClasses(AuthorizationStack stack, List<File> classfiles, List<File> jarfiles) {
         IAuthorizationPlugin pf;
+        
         for (File file : classfiles) {
             String classname = getClassName(file);
             if (classname.isEmpty()) {
                 continue;
             }
-            // load the class in memory and try to find a configured space for this class
-            if ((pf = handleLoadClass(classname)) != null && !stack.setPlugin(pf)) {
-                // if there is not configured space -> append it to the stack
-                addPlugin(stack, pf);
+            // Load the class in memory and try to find a configured space for this class.
+            if ((pf = handleLoadClass(classname)) != null) {
+                stack.setPlugin(pf);
```

Comment:
maybe some log that this plugin is not configured in the stack?

Reply:
If I do this:

```
             // Load the class in memory and try to find a configured space for this class.
             if ((pf = handleLoadClass(classname)) != null) {
                 stack.setPlugin(pf);
+            } else {
+                LOGGER.log(Level.INFO, "plugin {0} is not configured in the stack",
+                        classname);
             }
         }
```

then this will also match classes like `opengrok.auth.plugin.ldap.LdapFacade$ContentAttributeMapper` etc.